test/review4-regressions.test.mjs

```
@@ -82,6 +83,84 @@ test('resetCanvas clears liquify in-progress state', () => {
+  };
+
+  await saveSystem.autoSave(true);
+  assert.equal(writes.length, 0);
+
+  await saveSystem.autoSave(true, { forceWrite: true });
+  assert.equal(writes.length, 1);
+});
+
+test('liquify overlay buttons support keyboard click activation', () => {
+  const source = readFileSync(new URL('../src/js/pendefine/liquify.js', import.meta.url), 'utf8');
+  const match = source.match(/setupOverlayEvents\(\) \{(?<body>[\s\S]*?)\n  \}\n\n  showOverlay/);
```

ESLint

[error] Spaces are hard to count. Use {2}.

(no-regex-spaces)